server/eventHandlers/entityArchival.ts

```diff
@@ -18,6 +18,8 @@ import { IWebsocketsPayload, WebsocketMessageType } from '../../shared/types';
 const log = factory.getLogger(formatFilename(__filename));
 
 export default class extends IEventHandler {
+  private _name = 'Entity Archival';
```

Comment:
Why not make these `public readonly` since they're never being changed?

Reply:
You are correct - fixed in most recent commit